src/main/java/com/microsoft/azure/spring/data/documentdb/DocumentDbFactory.java

```diff
@@ -15,7 +15,7 @@
 
 public class DocumentDbFactory {
 
-    private static final String USER_AGENT_SUFFIX = "spring-data/" + PropertyLoader.getProjectVersion();
+    private static final String USER_AGENT_SUFFIX = Constants.USER_AGENT_SUFFIX + PropertyLoader.getProjectVersion();
```

Comment:
Shall we name it as USER_AGENT_PREFIX? And rename the old variable `USER_AGENT_SUFFIX` as `USER_AGENT`?

Reply:
OK, I thought the USER_AGENT_SUFFIX contains two parts.

Reply:
This is the fixed String defined in spring data -> org.springframework.data.repository.config.AnnotationRepositoryConfigurationSource.getRepositoryImplementationPostfix.java.